stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -454,6 +456,9 @@ public class MessageListView : ConstraintLayout {
 
         configureMessageOptions(tArray)
         tArray.recycle()
+        if (background == null) {
+            setBackgroundColor(ContextCompat.getColor(context, R.color.stream_ui_white_snow))
```

Comment:
This way you're changing the background of the parent ConstraintLayout right? I don't thinkg that binding the background of both views should be done... If that's the case, we could simply never set the background for this view and it would simply propagate the background of the container.